src/renderers/PartialViewRenderers/region/facilitiesInfo.js

```diff
@@ -228,3 +241,11 @@ function deleteFacility_handler() {
         })
     })
 }
+
+function facilityFilter_handler(){
+    $('input[type=radio][name=functionalFacilitiesDisplay]').on('change', e => {
```

Comment:
Same here, no magic strings

Reply:
Wait, is this HTML?

Reply:
Yes, that's jQuery selector string.

Reply:
Hm, not sure what the best practice is for HTML strings (whether they should be extracted). Will find out later, for now, resolved.